zerver/tests/test_templates.py

```diff
@@ -181,6 +181,7 @@ def get_context(self, **kwargs: Any) -> Dict[str, Any]:
             request=RequestFactory().get("/"),
             invite_as={"MEMBER": 1},
             max_file_upload_size = 25,
+            avatar_urls="www.zulip.com",
```

Comment:
I am not sure if this is correct, but the `zerver.tests.test_templates` was failing without this. @synicalsyntax @timabbott

Reply:
You are right, I changed to dict. But I'm not that comfortable writing a test for this.